AddOns/BasicFinancials/app/src/page/AssistedSetupBF.Page.al

```diff
@@ -0,0 +1,179 @@
+page 20600 "Assisted Setup BF"
+{
+    Caption = 'Basic Assisted Setup guide';
+    DeleteAllowed = false;
+    InsertAllowed = false;
+    LinksAllowed = false;
+    PageType = NavigatePage;
+    ShowFilter = false;
+
+    layout
+    {
+        area(content)
+        {
+            group(MediaStandard)
+            {
+                Caption = '';
+                Editable = false;
+                Visible = TopBannerVisible;
+
+                field(MediaResourcesStandard; MediaResourcesStandard."Media Reference")
+                {
+                    ApplicationArea = All;
+                    Editable = false;
+                    ShowCaption = false;
+                }
+            }
+
+            group(FirstPage)
+            {
+                Caption = '';
+
+                group("Welcome")
+                {
+                    Caption = 'Welcome';
+
+                    group(Introduction)
+                    {
+                        Caption = '';
+                        InstructionalText = 'The Basic extension enables the subset Dynamics 365 Business Central capabilities provided by the Basic license.';
```

Comment:
The Basic Experience extension enables the subset of Business Central capabilities provided by the Basic license.